### ISSUE TITLE
Request Fails

### ISSUE THREAD

**Issue by EZarnosky** [DeMille/headphones-for-musicbrainz#1]
When i try to add an artist or album the request fails. I am not sure if this has to do with the options screen foring everything to uppercase (I did reset my password to all upper and it still does not work). How about adding a test connection option to the extension? Or maybe more detail to the error the exteion is receiving - no connection, incorrect password... etc.

**Comment by DeMille** [comment]
Great idea, I just added a test button and pushed it to the chrome store. I changed the uppercase css rule too, that was pretty dumb huh

I realized you actually don't need to use your headphones user/pass at all when you have an API key.  I have headphones behind nginx w/ some basic auth, so that's what I was using the user/pass fields for.  Really those fields only are needed if you have headphones behind a reverse proxy with its own auth.  (I put a note of that on the options page too)

I'm not really sure whats happening with your requests though, can you tell me what the developer console in chrome says is the error?

One thing that does happen: when you add an album from an artist that headphones hasn't finished scraping it spits back a 500 error immediately.  (The headphones API is still a little rough.)  Once it finally loads the artist info it will automatically add the album like normal, as if it never gave an error.  Usually it just takes headphones a little while to process everything, and if you come back and refresh the page later youll see that everything was added.  I don't know if that's what you're seeing, or is it not even getting that far?

**Comment by EZarnosky** [comment]
Well I actually got the previous version working, not sure exactly how. I reset the password several times and the extension finally connected.

I removed the previous version and installed the new one. After a couple issues I got it connected. seems the API key needs to be lower case (I copied it from the old version which put it all in caps). The Test button IDed the issue correctly... Bad API Key.

I would say the issue is resolved. Thanks for the quick fix and update!
